hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/freon/TestRandomKeyGenerator.java

```diff
@@ -62,6 +62,7 @@ public static void init() throws Exception {
     raftClientConfig.setRpcRequestTimeout(Duration.ofSeconds(3));
     raftClientConfig.setRpcWatchRequestTimeout(Duration.ofSeconds(3));
     conf.setFromObject(raftClientConfig);
+    conf.setBoolean("hdds.datanode.chunk.data.validation.check", true);
```

Comment:
I will suggest that lets enable this flag by default on all the unit test via MiniOzoneCluster and also docker based tests.